src/components/data-management/FileUploadModal.jsx

```diff
@@ -169,6 +169,14 @@ const FileUploadModal = (props) => {
     </>
   );
 
+  const getFilePathToDisplay = (fileObject) => (
+    // if the file has a path, trim to just the file and its folder.
+    // otherwise simply use its name
+    fileObject.path
+      ? _.trim(Object.values(getFileSampleAndName(fileObject.path)).join('/'), '/')
+      : fileObject.name
```

Comment:
why don't all samples behave the same? why some have no name and we have to use the path?

Reply:
> why some have no name and we have to use the path

It's the other way around, for some fileObject's we don't have the `path` and have to use the `name`

I don't know why for some `fileObject`'s we use their name and some not. This behavior is copied over from the way the `file.name` is set because I had removed the file.name. This is the bit of code I copied:
<img width="754" alt="image" src="https://github.com/hms-dbmi-cellenics/ui/assets/8465903/343036b0-0e11-4812-916a-e91ab60623c5">

I am thinking that it might be a matter of how a file is chosen: if drag and drop you get the `path`, if select a file perhaps not. I'll check and get back to you on this one

Reply:
I tried using the selector and drag and drop on every technology type and none of them seemed to have a case with .path not set so I ended up leaving the function with only

`return _.trim(Object.values(getFileSampleAndName(fileObject.path)).join('/'), '/')`